README.md

```diff
@@ -13,27 +13,57 @@ npm install --save-dev @faker-js/cli
 ## Usage
 
 ```
-faker module_name method_name
+npx faker <moduleName> <functionName>
```

Comment:
I'm not sure whether you should call this placeholder `moduleName` or just `module` (same for `functionName`).
For me, it would be clear even without the name and the name suffix feels redundant.

Reply:
Since I did not figure out what the best approche for documentation would be. Thats why I wanted to be as explicit as possible. I might change this in the future, but the current naming feels more safe for me.